noble/assetlist.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "noble",
+  "assets": [
+    {
+      "description": "The permissioned staking asset for Noble Chain",
+      "denom_units": [
+        {
+          "denom": "ustake",
+          "exponent": 0,
+          "aliases": [
+            "microstake"
+          ]
+        },
+        {
+          "denom": "stake",
+          "exponent": 6
+        }
+      ],
+      "base": "ustake",
+      "name": "Stake",
+      "display": "stake",
+      "symbol": "STAKE"
+    },
+    {
+      "description": "Frienzies are an IBC token redeemable exclusively for a physical asset issued by the Noble entity.",
+      "denom_units": [
+        {
+          "denom": "ufrienzies",
+          "exponent": 0,
+          "aliases": [
+            "microfrienzies"
+          ]
+        },
+        {
+          "denom": "frienzies",
+          "exponent": 6
+        }
+      ],
+      "base": "ufrienzies",
+      "display": "ufrienzies",
```

Comment:
I assume display denom should be 'frenzies' (the unit with 6 exponent

Reply:
https://github.com/strangelove-ventures/chain-registry/commit/c5fa1f2c8d69515ffec1cde46dc910327e66706f